### ISSUE TITLE
Ability to drag and/or reposition the overlays how we like

### ISSUE THREAD

**Issue by burritosoftware** [designeerlabs/discord-vr#11]
I'd love if I could position the overlay in the world right above me, but it seems stuck to the floor.

**Comment by designeerlabs** [comment]
Yep, the attach to world was just to show that it can be done. The reposition part is planned for the next update.
